src/_playground/App.scss

```diff
@@ -73,10 +73,29 @@ $fd-fonts-path: '../../node_modules/fiori-fundamentals/dist/fonts/';
             overflow-y: scroll;
             background-color: white;
             padding: 2rem;
+          
+            .frDocs-Content__githubLogo {
+                float: right;
+                z-index: 100;
+                cursor: pointer;
+                width: 2rem;
+                height: 2rem;
+                font-size: 1rem;
+
+                svg {
+                    width: 2rem;
+                    height: 2rem;
+                }
+            }
 
             .frDocs-Content__header, .frDocs-markdown > h1 {
                 margin: 0 0 12px;
                 font-size: 2.2rem;
+                line-height: 2.2rem;
+            }
+
+            .frDocs-Content__header:empty {
+                display: none;
```

Comment:
The display looks good now.  My only remaining question is what is the purpose of this `:empty` block?  If I remove it, the logo still appears correct.

Reply:
On the homepage, since there is no header text for the `<h1>` tag, the logo will be misaligned (see below). The first title on homepage comes from the markdown.

<img width="1430" alt="screen shot 2019-02-07 at 23 10 59" src="https://user-images.githubusercontent.com/6461288/52449002-a7c8fb00-2b2d-11e9-99f1-6617c34b96e0.png">


By using the `:empty` we can hide the `<h1>` and the space it takes on the homepage only (when empty) and have the logo line up with the markdown title.

<img width="1433" alt="screen shot 2019-02-07 at 23 06 15" src="https://user-images.githubusercontent.com/6461288/52448864-2d987680-2b2d-11e9-9321-7ab9ce34f9ed.png">

Reply:
Ok, I see now.  Actually, there are two `<h1>` tags -- that's what was confusing me.  The empty one with the class `frDocs-Content__header` and then the `<h1>` that gets pulled in from the markdown file (with no class on it).

Reply:
Yep exactly, so we're hiding the empty `frDocs-Content__header`